### ISSUE TITLE
DotNetCoreCLIV2: Restore-Command is missing option 'runtime'

### ISSUE THREAD

**Issue by JohannesDrexler** [microsoft/azure-pipelines-tasks#8643]
## Environment
- Server - Azure Pipelines or TFS on-premises?
   AzurePipelines

- Agent - Hosted or Private: 
   Private, 2.140.2 running on win2012r2, also 2.140.2 running on win2016

## Issue Description

.Net Core task allows custom commands and some predefined like 'restore'. Problem is i have to restore using a certain runtime but also use packages from a private AzureArtifacts-Feed. When i select restore as command the authentication gets handled correctly but i can't set the runtime. when i select custom, i can configure restore using the runtime i need but authentication is not handled.

In short:
I need an option to set the runtime when selecting 'restore' as command.

**Comment by amaljg** [comment]
@JohannesDrexler ,

As you noted, this is currently not supported by the task. I'm adding @pulkitaggarwl to consider this feedback in a future version of the task.

**Comment by elbatk** [comment]
Hello! I'm a PM over on the Azure Artifacts team. I wanted to drop in here and address some of these issues with the news that we're currently reevaluating the types of tasks we offer in Azure Pipelines, and looking to focus more on lighter-weight tasks that cover authentication and setup. In lieu of the task redesign, we're going to forego making changes to the current tasks (with a few exceptions). We hope the new tasks we spin up in the next quarter or so are easier and more helpful, be on the lookout!

**Comment by bender2k14** [comment]
> We hope the new tasks we spin up in the next quarter or so are easier and more helpful, be on the lookout!

Hello @elbatk.

Q3 has come and gone.  I haven't noticed any changes, but maybe they happened and I just missed them.

Can you tell us where we should we be looking?

Thanks :)

**Comment by bender2k14** [comment]
Meta:
> @bender2k14 unassigned @infin8x 1 minute ago

I did what now?
